src/components/form/HdRadioCard.vue

```diff
@@ -0,0 +1,280 @@
+    },
+    /** @returns {string} */
+    inputName() {
+      return `${this.name}-${this.nativeValue}`;
+    },
+  },
+  methods: {
+    validateForm() {
+      if (this.required && !this.value) return this.t.FORM.VALIDATION.REQUIRED;
+
+      const firstFailingRule = this.customRules.find(({ validate }) => !validate(this.value));
+      if (firstFailingRule) return firstFailingRule.errorMessage;
+
+      return null;
+    },
+    /**
+     * This is a default validate function
+     * it is a special function that is automatically
+     * invoked by HdForm.
+     *
+     * By surrounding HdRadioCard with a HdRadioCardGroup
+     * this function is automatically overriden.
+     *
+     * @returns {boolean}
+     */
+    validate() {
+      this.error = this.validateForm();
+      return !this.hasValidationErrors;
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import "homeday-blocks/src/styles/mixins.scss";
+
+::v-deep path {
+  transition: fill ($time-s * 2);
+}
+
+input[type="radio"] {
+  display: none;
+}
+
+.card {
+  position: relative;
+  cursor: pointer;
+  border: 1px solid getShade($neutral-gray, 60);
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: $sp-m;
+  transition: box-shadow ($time-s * 2), border ($time-s * 2);
+  @include elevation(0);
+}
+
+.card--selected {
+  @include elevation(0);
+}
+
+label:hover > .card,
+label:active > .card {
+  @include elevation(2);
+}
+
+label:focus > .card {
+  box-shadow: 0px 0px 14px getShade($dodger-blue, 110), 0px 0px 1px getShade($dodger-blue, 110);
+}
+
+.card--disabled,
+label:hover > .card--disabled,
+label:active > .card--disabled,
+label:focus > .card--disabled {
+  cursor: not-allowed;
+  border-color: getShade($quaternary-color, 60);
+  border-width: 1px;
+  @include elevation(0);
+}
+
+.card--disabled > .card__icon ::v-deep svg {
+  opacity: 0.4;
+}
+
+.card--disabled > .card__icon ::v-deep path {
+  filter: grayscale(0.95);
+}
+
+.card__border {
+  position: absolute;
+  inset: -1px;
+  border-radius: 4px;
+  transition: border ($time-s * 2);
+  border: 4px solid transparent;
+}
+
+.card--selected:not(.card--disabled) > .card__border,
+label:focus > .card:not(.card--disabled) > .card__border {
+  border-color: getShade($dodger-blue, 110);
+}
+
+label:active > .card:not(.card--disabled)  > .card__border {
+  border-width: 2px;
+  border-color: getShade($dodger-blue, 110);
+}
+
+.card__control-label {
+  margin-top: $sp-xs;
+  color: $primary-color;
+  @include font("DS-100");
+}
+
+.card__indicator {
+  position: absolute;
+  top: $sp-m;
+  right: $sp-m;
+}
+
+.card--tile {
+  flex-direction: row;
+  padding: $sp-s;
+}
+
+.card--tile > .card__indicator {
+  position: relative;
+  top: unset;
+  right: unset;
+  align-self: center;
+  order: 1;
+}
+
+.card--tile > .card__control-label {
+  flex: 1;
+  margin-top: 0;
+  margin-left: $sp-xs;
+  margin-right: $sp-xs;
+}
+
+.card--tile ::v-deep svg {
+  max-width: 44px;
```

Comment:
Should this be relative to the font size? We have a `rem-calc` function if it is needed

Reply:
That's a good catch!

5d1a72c